src/responder.ts

```diff
@@ -443,13 +465,13 @@ export class EthereumResponderManager {
                 // Should we keep inactive responders anywhere?
                 this.responders.delete(responder);
             })
-            .on(ResponderEvent.AttemptFailed, (responseFlow: ResponseFlow, doh, attemptNumber) => {
+            .on(ResponderEvent.AttemptFailed, (responseFlow: ResponseFlow, doh, attemptNumber: number) => {
```

Comment:
doh doesnt fail here because on() accepts ...args: any[]? We should probably insist that it's of type error anyway, since it would be a bug if we were throwing an object that wasnt.